lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -11,6 +14,7 @@
 from ..test_utils import SocialFacebookTestCase
 
 
+@patch.dict(settings.FEATURES, {"CUSTOM_COURSES_EDX": False})
```

Comment:
Was this change necessary?  I'm surprised that we have CCX related tests in this file.